controllers/vgmanager.go

```diff
@@ -78,8 +78,7 @@ func (v vgManager) ensureCreated(r *LVMClusterReconciler, ctx context.Context, l
 		ds.Spec.Template.Spec.Volumes = dsTemplate.Spec.Template.Spec.Volumes
 
 		// service account
-		// TODO(rohan) get from env
-		ds.Spec.Template.Spec.ServiceAccountName = "lvm-operator-vg-manager"
+		ds.Spec.Template.Spec.ServiceAccountName = VGManagerServiceAccount
```

Comment:
Please move these changes to the "add vgmanager..." commit. This commit should have only the yaml changes.

Reply:
done.